deploy/charts/external-secrets/values.schema.json

```diff
@@ -659,6 +659,9 @@
                                 },
                                 "renewBefore": {
                                     "type": "string"
+                                },
```

Comment:
CI was initially failing, so this was added as part of `make reviewable`

Reply:
I don't understand why this wasn't picked up when I was merging that pr. 🤔 Thank you for adding it.